src/utils/etherspot.js

```diff
@@ -284,3 +285,37 @@ export const getChainTokenListName = (chain: Chain): ?string => {
 
   return null;
 };
+
+/**
+ * How use this function
+ * @param {*} assets is mendatory. like own tokens, default tokens
+ * @param {*} chain is optional. using chain to find perticular chain assets
+ * @param {*} stableTokens is optional. check assets in this tokens availability
+ * @param {*} filteringAssets is optional. mostly default token list or default stable list.
+ * @returns if assets is null then return blank array. otherwise return filter tokens list
+ */
+export const filteredChainAssets = (
+  assets: AssetsPerChain,
+  chain?: Chain,
+  stableTokens?: Asset[],
+  filteringAssets?: Asset[],
```

Comment:
Maybe change the name of `filteringAssets`? This sounds like like a process or something rather than a variable - maybe `assetsToFilter`?